storage/src/state/tests.rs

```diff
@@ -346,3 +354,34 @@ fn test_transaction_fees() {
     assert_eq!(new_coinbase_record.value(), expected_block_reward);
     assert_eq!(output_record.value(), amount);
 }
+
+#[test]
+fn test_get_all_ciphertexts() {
+    let rng = &mut thread_rng();
+    let terminator = AtomicBool::new(false);
+    // Initialize a new ledger.
+    let ledger = create_new_ledger::<Testnet2, RocksDB>();
+
+    // Initialize a new account.
+    let account = Account::<Testnet2>::new(&mut thread_rng());
+    let address = account.address();
+
+    // Mine the next block.
+    let (block, _) = ledger
+        .mine_next_block(address, true, &[], &terminator, rng)
+        .expect("Failed to mine");
+    ledger.add_next_block(&block).expect("Failed to add next block to ledger");
+
+    let expected_ciphertexts_set = to_hash_set_test::<Testnet2>(
+        ledger
+            .get_blocks(0, ledger.latest_block_height())
+            .unwrap()
+            .iter()
+            .flat_map(|block| block.commitments())
+            .map(|commitment| ledger.get_ciphertext(commitment).unwrap())
+            .collect::<Vec<_>>(),
```

Comment:
```suggestion
            .collect::<HashSet<_>>(),
```
(which renders the call to `to_hash_set_test` redundant)